iree/compiler/Dialect/HAL/Conversion/StreamToHAL/test/cmd_ops.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: iree-opt -split-input-file -iree-hal-conversion %s | IreeFileCheck %s
+
+// TODO(#7277): swap HAL Conversion->Conversion and add tests.
```

Comment:
can you update these TODOs? (looks like the find/replace for Conversion2 affected them)

(of course, adding the tests would be nice too, but that can wait ;P)